portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,29 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+function genAlbums() {
+    var past = new Set();
+    
+    const albumNames = ['2014 Forest Hill Drive - J.Cole', 'ASTROWORLD - Travis Scott', 'Blonde - Frank Ocean', 'Currents - Tame Impala', 'Dont Forget About Me - Dominic Fike', 'GINGER - BROCKHAMPTON', 'Lyrics To Go Vol.1 - Kota the Friend', 
```

Comment:
When an array has a lot of entries, its better to write it like this
```
const albumNames = ['2014 Forest Hill Drive - J.Cole',
                   'ASTROWORLD - Travis Scott',
                   'Blonde - Frank Ocean',
                   ....
                   ....
```
Its more readable

Reply:
Agreed, in Google we have a limit of 100 chars per line.